### PR TITLE
Do not ship ostree repo type as rpm

### PR DESCRIPTION
**Pull request by jlsherrill**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Corresponds to https://github.com/theforeman/puppet-katello/pull/237

### CODE REVIEW

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
Well, it tested this correctly, however, we need to update the package_manifest for these special packages.

**Comment by ehelms** [comment]
@jlsherrill Mind applying this patch with your changes:

```
diff --git a/package_manifest.yaml b/package_manifest.yaml
index e20fab3..86a0224 100644
--- a/package_manifest.yaml
+++ b/package_manifest.yaml
@@ -263,7 +263,10 @@ katello_packages:
     rubygem-hammer_cli_foreman_virt_who_configure: {}
     rubygem-hammer_cli_katello: {}
     rubygem-jenkins_api_client: {}
-    rubygem-katello: {}
+    rubygem-katello:
+      build_package_tito_releaser_args:
+        - "--arg jenkins_job=test_katello_core"
+        - "--arg jenkins_job_id=lastSuccessfulBuild"
     rubygem-mixlib-shellout: {}
     rubygem-qpid_messaging: {}
     rubygem-robotex: {}
```

That way we can test that they work and your build will pass.

**Comment by ehelms** [comment]
[test rpm]

File: package_manifest.yaml [hunk 1]
```diff
@@ -263,7 +263,10 @@ katello_packages:
     rubygem-hammer_cli_foreman_virt_who_configure: {}
     rubygem-hammer_cli_katello: {}
     rubygem-jenkins_api_client: {}
-    rubygem-katello: {}
+    rubygem-katello:
+      build_package_tito_releaser_args:
```
Comment (ehelms) [inline package_manifest.yaml]: One more thing (thanks for being the guinea pig), add this under `rubygem-katello`:

```
releasers:
     - koji-katello-jenkins
```

**Comment by ekohl** [comment]
@ehelms we probably need to add those options to a few more nightly packages.

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
Thanks @jlsherrill

**Pull request merged**